txtdirect.go

```diff
@@ -336,7 +341,7 @@ func Redirect(w http.ResponseWriter, r *http.Request, c Config) error {
 		if err != nil {
 			return err
 		}
-		reverseProxy := proxy.NewSingleHostReverseProxy(u, "", proxyKeepalive, proxyTimeout, fallbackDelay)
+		reverseProxy := proxy.NewSingleHostReverseProxy(u, "", proxyKeepalive, proxyTimeout)
```

Comment:
This breaks with the latest Caddy master branch.

Reply:
I thought I had discarded this change. Will bring it back.